tests/src/Traits/HttpRequestTrait.php

```diff
@@ -4,6 +4,7 @@
 
 use Drupal\graphql\Entity\Server;
 use Symfony\Component\HttpFoundation\Request;
+use Drupal\graphql\Entity\ServerInterface;
```

Comment:
cool, what is usually done in the list of used namespaces is, that they are ordered alphabetically, so it's easier for a developer to look through the list. sometimes the list is really huge and if we are checking some namespaces it's harder to go through unordered list.

Hint: your IDE will most certainly help you with that. Almost every IDE is able to add a namespace with some shortcut to the correct position. If you are using PHPstorm you can try it yourself. Just delete the namespace here and go the method which is using it and click on it. Then you can use a shortcut `Alt + Enter` which pops up a dialog where you choose `Import class`:
![phpstorm-import-class-dialog](https://user-images.githubusercontent.com/5792505/89891004-9b1adc80-dbd4-11ea-82ad-8b8d35f8c731.png)

You click on it and just select proper class from the list:
![phpstorm-class-selection](https://user-images.githubusercontent.com/5792505/89890918-7888c380-dbd4-11ea-8206-d06b19db2388.png)

And boom the namespace is added to the top of the file to the correct position :)